cavityDisplayGUI.py

```diff
@@ -96,13 +98,19 @@ def __init__(self, parent=None, args=None):
                     # .add_callback is called when statusPV changes value
                     statusPV.add_callback(partial(self.statusCallback, cavityWidget))
 
+                    # .add_callback is called when descriptionPV
+                    descriptionPV.add_callback(partial(self.descriptionCallback, cavityWidget))
+
     # Updates shape depending on pv value
     def severityCallback(self, cavity_widget, value, **kw):
         self.changeShape(cavity_widget,
                          SHAPE_PARAMETER_DICT[value]
                          if value in SHAPE_PARAMETER_DICT
                          else SHAPE_PARAMETER_DICT[3])
 
+    def descriptionCallback(self, cavity_widget: QWidget, value, **kw):
```

Comment:
Looking at this, we can probably make it a static method

Reply:
What's a static method? Is that no longer a callback? Happy to make the change but unsure how to do it since I don't know what a static method is!

Reply:
It's the `@staticmethod` decorator that tells us this function doesn't need "self"

Reply:
Wow, you even told me in this comment that this would no longer need "self" if I made it a staticmethod!